README.md

```diff
@@ -100,6 +100,71 @@ Options:
   -h, --help                       output usage information
 ```
 
+### Version 2
+
+There currently exist two deployment options while we phase into complete use of version 2. A version 1 deployment appends a file's checksum to the filename itself, allowing releases of individual files. Version 2 deploys all assets under a single common S3 bucket prefix which is the hash of both the git commit and branch to which those assets pertain.
+
+Version 2 also introduces a few new configuration changes and overall deployment workflow:
+
+* You now have the ability to set an environment to master-only, meaning you'll get a warning and additional confirmation prompt if you attempt to deploy non-master branch assets.
+
+```json
+{
+  "key": "production",
+  "ssmKey": "/production/asset-checksum",
+  "serverEnvironment": "production",
+  "label": "Production",
+  "slackChannel": "#deployments",
+  "masterOnly": true,
+  "v2": true
+}
+```
+
+* You must also (temporarily) add the `v2` key to any environment with which you wish to use version 2. This requirement will be removed once a full cutover to version 2 is made.
+
+* You can (and must) use file globs to describe which directories and/or files you'd like gurgler to deploy.
+
+```json
+"localFileGlobs": [
+  {
+    "pattern": "build/*",
+    "ignore": [
+      "*/garbage_file.json"
+    ]
+  }
+],
+```
+
+* Version to introduces a new CLI command `configure`. Use it to build a `gurgler.json` in the project root. Webpack can use this file as shown in the following example to know how to build internal references to other files in the build directory.
```

Comment:
I think I get this _better_ now but I'm still a bit confused. Do I only need to run `configure` once? Or should I not need to run it "day to day"?

Reply:
I should clarify this a bit in the README.md. You'd want to run `configure` prior to every build. That way webpack knows what the current version is and can append it to the public paths of any internal file references. In `ui-components` I have a script that combines `configure` and `build` commands so you don't have to worry about it.

Reply:
Cool, as long as the README lays out how you should use the system in the most common uses cases.